### PR TITLE
Externalize Javascript: finish angular pages

### PR DESCRIPTION
**Pull request by orangejenny**
@czue / @millerdev 

https://github.com/dimagi/js-guide/blob/master/migrating.md#angular

### CODE REVIEW

File: corehq/apps/data_interfaces/templates/data_interfaces/add_automatic_update_rule.html [hunk 1]
```diff
@@ -5,96 +5,13 @@
 {% load crispy_forms_tags %}
```
Comment (orangejenny) [inline corehq/apps/data_interfaces/templates/data_interfaces/add_automatic_update_rule.html]: @gcapalbo Is this page dead code? I had to add a url for `AddAutomaticUpdateRuleView` to test it.

**Review by millerdev — APPROVED** [review]

**Review by czue — APPROVED** [review]
:+1: pending response from @gcapalbo

File: corehq/apps/data_interfaces/templates/data_interfaces/add_automatic_update_rule.html (hunk 1 above)
Comment (gcapalbo) [reply corehq/apps/data_interfaces/templates/data_interfaces/add_automatic_update_rule.html]: it's not used anymore, it's leftover from when these had slightly different models. it's on my list to remove that page and the old models

**Review by gcapalbo — APPROVED** [review]

File: corehq/apps/data_interfaces/templates/data_interfaces/add_automatic_update_rule.html (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/data_interfaces/templates/data_interfaces/add_automatic_update_rule.html]: Cool, I'll leave removing them to you.

**Pull request merged**